examples/trl_example/stack_llama_2/README.md

```diff
@@ -0,0 +1,36 @@
+# DPO pipeline for the creation of StackLlaMa 2: a Stack exchange llama-v2-7b model
+
+## Prerequisites
+
+Install all the dependencies in the `requirements.txt`:
+
+```
+$ pip install -U -r requirements.txt
+```
+
+## Training
+
+There were two main steps to the DPO training process:
+1. Supervised fine-tuning of the base llama-v2-7b model to create llama-v2-7b-se:
+    - `python ../../gaudi_spawn.py --world_size 8 --use_mpi sft_llama2.py --training_args.output_dir="sft_output" --training-args.report_to none`
+1. Run the DPO trainer using the model saved by the previous step:
+    - `python ../../gaudi_spawn.py --world_size 8 --use_mpi dpo_llama2.py --model_name_or_path="sft_output/final_merged_checkpoint" --output_dir="dpo_output" --report_to=none`
+
```

Comment:
what about Merging the adaptors? have you tried it?

Reply:
merged adapter are in SFT part. see https://github.com/huggingface/trl/blob/main/examples/research_projects/stack_llama_2/scripts/sft_llama2.py#L183C1-L187C66